phet/info.json

```diff
@@ -7,5 +7,6 @@
   "zim_file": "content.zim",
   "embed_zim": false,
   "ic_launcher": "icon.png",
+  "ic_launcher_master": "icon_master.png",
```

Comment:
phet/icon_master is an entire screen, not just a logo

Reply:
Is a stupid error, will fix it.